Milestone 7/script.js

```diff
@@ -0,0 +1,264 @@
+// UI Elements 
+const isButton = document.querySelector('.is-button')
+const result = document.querySelector('.result')
+const userInput = document.querySelector('.user-input')
+const spinnerElement = document.querySelector('#spinnerElement')
+const errorMessage = document.querySelector('.error-message')
+const resultsContainer = document.querySelector('.results-container')
+const checkBox = document.querySelector('#check-box')
+const form = document.querySelector('.form')
+const select = document.querySelector('select')
+
+
+// Hide Error on Default 
+hideError()
+
+
+// Event Listerners
+// isButton.addEventListener('click', calcFibo)
+
+
+// Hide Spinner On Default
+hideSpinner()
+
+
+
+// Fucntions    // Calc Fibo Main Fucntion
+
+
+function calcFibo(e) {
+
+    if (userInput.value === '') return
+    userInput.style.borderColor = 'black'
+    hideError()
+    const number = userInput.value
+
+    if (number > 50) {
+        showError()
+        return
+    }
+
+    showSpinner()
+
+
+
+
+    // Fetch data from local Server
+    fetch(`http://localhost:5050/fibonacci/${number}`)
```

Comment:
You should define the server url as a const before the function, that way it is only created once, and then you just add the number accordingly